pkg/agent/multicast/mcast_controller.go

```diff
@@ -270,6 +279,11 @@ func (c *Controller) Initialize() error {
 		klog.ErrorS(err, "Failed to install multicast initial flows")
 		return err
 	}
+	c.queryGroupId = c.v4GroupAllocator.Allocate()
+	klog.Infof("multicast/mcast_controller: c.queryGroupId=%v", c.queryGroupId)
+	if c.anpEnabled {
+		c.mcastValidator.Initialize(c.queryGroupId)
```

Comment:
Initialize should be called inside mcastValidator?

Reply:
removed